File: Practica_1/src/gestorAplicacion/Persona.java
package gestorAplicacion;

abstract public class Persona {
    protected String nombre;
    protected int identificacion;
    protected int telefono;

    // METODOS

    public abstract String obtenerInformacion();

    // CONSTRUCTOR
    public Persona(String nombre, int identificacion, int telefono) {
        this.nombre = nombre;
        this.identificacion = identificacion;
        this.telefono = telefono;
    }

    // GETTERS AND SETTERS
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getIdentificacion() {
        return identificacion;
    }

    public void setIdentificacion(int identificacion) {
        this.identificacion = identificacion;
    }

    public int getTelefono() {
        return telefono;
    }

    public void setTelefono(int telefono) {
        this.telefono = telefono;
    }

}

File: practica2-grupo2-Equipo3/src/gestorAplicacion/modelos/Persona.py
import abc

class Persona(abc.ABC):
    def __init__(self, nombre, identificacion, telefono):
        self.nombre = nombre
        self.identificacion = identificacion
        self.telefono = telefono

    @abc.abstractmethod
    def informacion(self):
        pass

    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre

    def getIdentificacion(self):
        return self.identificacion

    def setIdentificacion(self, identificacion):
        self.identificacion = identificacion

    def getTelefono(self):
        return self.telefono

    def setTelefono(self, telefono):
        self.telefono = telefono
